src/services/profiles/groups.ts

```diff
@@ -0,0 +1,100 @@
+import type { HttpInstance } from '../../types';
+import { AffectedRecords, ObjectId } from '../types';
+import { Group } from './types';
+
+export default (client, httpAuth: HttpInstance) => ({
+  /**
+   * Add a group enlistment to a profile
+   * Permission | Scope | Effect
+   * - | - | -
+   * none | | Add a group enlistment for your profile only
+   * `ADD_PATIENT` | `staff enlistment` | Add a group enlistment for any profile of this group
+   * `ADD_PATIENT` & `ACTIVATE_PRESCRIPTIONS` | `global` | Add a group enlistment for any profile for any group
+   *
+   * @param profileId Id of the targeted profile
+   * @param requestBody Group data
+   * @returns Group
+   * @throws {ResourceAlreadyExistsError}
+   * @throws {ResourceUnknownError}
+   */
+  async addGroupToProfile(
```

Comment:
`sdk.profiles.groups.addToProfile` instead of `sdk.profiles.groups.addGroupToProfile` ?

Reply:
yeah agree! sounds better!